application-admintools-default/src/main/java/com/xwiki/admintools/script/AdminToolsScriptService.java

```diff
@@ -97,4 +121,38 @@ public String getFilesSection()
     {
         return this.adminToolsManager.getFilesSection();
     }
+
+    /**
+     * Check if an Admin Tools Health Check job for the wiki from where the request was made exists. If it does, return
+     * the job instance, else create a new Admin Tools health check request for the given wiki and start the execution.
+     *
+     * @return the asynchronous background job that will execute the request.
+     */
+    public Job runHealthChecks()
+    {
+        try {
+            List<String> requestId = this.getHealthCheckJobId();
+            Job job = this.jobExecutor.getJob(requestId);
+            if (job == null) {
+                HealthCheckJobRequest healthCheckJobRequest = new HealthCheckJobRequest(requestId);
+                return this.jobExecutor.execute(HealthCheckJob.JOB_TYPE, healthCheckJobRequest);
```

Comment:
It's not enough to check admin right in the Velocity templates and macros because any user with script right can create a new wiki page and call this script service to trigger the health check.